### PR TITLE
Cdmp 2364 lint nodejs code in drone pipeline with standard

### PR DESCRIPTION
**Pull request by nefischer**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -33,6 +33,14 @@ tropo-lint:
 node-test: ./lambda-src/node_modules/
 	cd lambda-src && npm test
 
+node-lint: ./lambda-src/node_modules/
+	./lambda-src/node_modules/.bin/standard lambda-src/*.js
+	./lambda-src/node_modules/.bin/standard --env mocha test/*.js
+
+node-lint-fix: ./lambda-src/node_modules/
+	./lambda-src/node_modules/.bin/standard --fix lambda-src/*.js
+	./lambda-src/node_modules/.bin/standard --fix --env mocha test/*.js
```
Comment (daniel-ac-martin) [inline Makefile]: Would be a bit nicer to have an npm script for running this. (As it sorts out the path and is a bit more idiomatic.)

e.g. `npm run lint`

That would also give us the option of side-stepping make in CI if that's more convenient.

Comment (daniel-ac-martin) [reply Makefile]: #9

**Comment by daniel-ac-martin** [comment]
Looks good otherwise.

File: Makefile (hunk 1 above)
Comment (nefischer) [reply Makefile]: PR accepted. Thanks @daniel-ac-martin

**Comment by nefischer** [comment]
closing this PR and opening one into master

**Pull request closed without merging**